bin/index.js

```diff
@@ -152,7 +158,11 @@ async function main(settings) {
   // Generate outputs
   generateSchemaFile(simplifiedSchema, settings);
   generateRequestFile(simplifiedSchema, settings);
-  generateAirtableFile(settings);
+
+  // Don't overwrite file if setting is false
+  if (settings.overwrite) {
```

Comment:
Nit: might be useful to log something to indicate whether the `airtable.js` file was overwritten as a sanity check/reminder when the script is run